node-notes-cli/app.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs');
+const dataObject = require('./data.json');
+
+const notes = dataObject.notes;
+if (process.argv[2] === 'read') {
+  for (const key in notes) {
+    console.log(`${key}: ${notes[key]}`);
+  }
+} else if (process.argv[2] === 'create') {
+  notes[`${dataObject.nextId}`] = process.argv[3];
+  dataObject.nextId = dataObject.nextId + 1;
+} else if (process.argv[2] === 'delete') {
+  delete notes[process.argv[3]];
+  dataObject.nextId--;
+  const notesStorage = [];
+  for (const key in notes) {
+    notesStorage.push(notes[key]);
+    delete notes[key];
+  }
+  let counter = 1;
+  for (let i = 0; i < notesStorage.length; i++) {
+    notes[`${counter}`] = notesStorage[i];
+    counter++;
+  }
```

Comment:
Once an id is used, it's used forever.

![image](https://user-images.githubusercontent.com/7432943/129278229-91b7bf12-e5bd-48a3-b2f6-7a5407de12c9.png)